### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by SuperFluffyGame** [community/community#37909] [GitHub Education]
I would like to sign up for the student developer pack, however I am using a non-school account.  I have tried adding the school email in settings but it asked for a verification code and my school blocks all out of domain emails. Is there a way to add it to this account without email verification?

**Comment by hongducdev** [comment]
You must ensure that you are over 13 years old to join the Student Developer Pack. You can access the link: https://education.github.com/pack/join. Please send your student card or other document proving you are a student. Github will confirm and let you join. Hope it can be of help to you. Good luck.

**Comment by samus-aran** [comment]
Hi @SuperFluffyGame 👋 

Welcome to the Community! Great question and often asked. 💭  Although the verification process prefers academic named emails, we understand that isn't always possible. Global Campus verification does allow non school emails but in some cases (depending on the selected school) it's required. Let us know how you get on. 👍
